R/db-mssql.R

```diff
@@ -0,0 +1,152 @@
+#' @export
+`db_desc.Microsoft SQL Server` <- function(x) {
+  info <- DBI::dbGetInfo(x)
+  host <- if (info$host == "") "localhost" else info$host
+
+  paste0("Microsoft SQL Server ", info$serverVersion, " [", info$user, "@",
```

Comment:
Can you please follow the tidyverse indenting rules, i.e. http://style.tidyverse.org/syntax.html#long-lines ?